packages/react/src/components/F0BigNumber/internal/defaults.ts

```diff
@@ -0,0 +1,7 @@
+import { NumericFormatterOptions } from "@/lib/numeric"
+
+export const defaultFormatterOptions: NumericFormatterOptions = {
+  locale: "en-US",
```

Comment:
We need to make sure we set this properly in the monorepo

Reply:
ok!

Reply:
We already are setting it in l10nContext, but i realized we were not using it a defauilt value, I created hook to manage that